### PR TITLE
Add missing parameter when monitoring OSP services

### PR DESCRIPTION
**Pull request by leifmadsen**
Add CollectdEnableSensubility: true to the stf-connectors.yaml when
enabling resource usage monitoring of OpenStack services.

Resolves: rhbz#1954155

### CODE REVIEW

**Comment by pleimer** [comment]
Can you add it to the stf-connectors.yaml mentioned in section '5.6.4. Creating the OpenStack environment file' as well?

File: doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file.adoc [hunk 1]
```diff
@@ -183,6 +183,8 @@ parameter_defaults:
           role: edge
           verifyHostname: false
           sslProfile: sslProfile
+
+    CollectdEnableSensubility: true  # <7>
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file.adoc:187]: Does the placement of this matter in the paramater defaults section?

Comment (leifmadsen) [reply doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file.adoc:188]: Does not matter the order

File: doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file.adoc [hunk 2]
```diff
@@ -183,6 +184,8 @@ parameter_defaults:
...
```
Comment (JoanneOFlynn2018) [inline doc-Service-Telemetry-Framework/modules/proc_creating-openstack-environment-file.adoc:188]: Two instances of the same parameter. Deleting this one as the other one is group with other Sensubility params

```suggestion
```

**Review by JoanneOFlynn2018 — APPROVED** [review]
Looks good Leif. I made 2 minor commits because we both made the same changes yesterday evening. Should be good to go! Will you take one final look over?

**Comment by leifmadsen** [comment]
Looks good. Merging!

**Pull request merged**